src/plotly-graph-card.ts

```diff
@@ -464,8 +472,7 @@ export class PlotlyGraph extends HTMLElement {
     const was = this.parsed_config;
     this.parsed_config = newConfig;
     const is = this.parsed_config;
-    if (!this.contentEl) return;
```

Comment:
I think this was meant to stay.

Reply:
It's not needed anymore:)
Now that the HTML is setup in the constructor, this is always defined